### PR TITLE
Alice Controller fix on demo

### PR DESCRIPTION
**Pull request by Maxino22**
#99 fix for Angular Alice controller.  updated the version of Angular CLI to version 16, modified Dockerfile to update node version npm version and forced some deprecated dependencies .

### CODE REVIEW

**Comment by swcurran** [comment]
Awesome — thanks.  @amanji — please do a quick review and see if we can merge.

File: AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Stage: build
-FROM node:14 as build
+FROM node:18 as build
 WORKDIR /app
 COPY package*.json /app/
-RUN npm install
+RUN npm install -g npm@10.7.0
```
Comment (amanji) [inline AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile:5]: Unsure why we need to pin to a specific npm version

File: AliceFaberAcmeDemo/controllers/alice-controller/src/styles.scss [hunk 2]
```diff
@@ -1,2 +1,3 @@
 /* You can add global styles to this file, and also import other style files */
-@import "~bootstrap/scss/bootstrap";
\ No newline at end of file
+// @import "~bootstrap/scss/bootstrap";
+@import "../node_modules/bootstrap/scss/bootstrap";
```
Comment (amanji) [inline AliceFaberAcmeDemo/controllers/alice-controller/src/styles.scss:3]: Is there a cleaner way to import without using relative paths to `node_modules`

**Review by amanji — COMMENTED** [review]
Just a couple of small questions. If they cannot be resolved, it's not a big deal and we can go ahead and merge this.

File: AliceFaberAcmeDemo/controllers/alice-controller/src/styles.scss (hunk 2 above)
Comment (amanji) [reply AliceFaberAcmeDemo/controllers/alice-controller/src/styles.scss:3]: I see it's done this way even in the bootstrap docs:

https://getbootstrap.com/docs/5.0/customize/sass/#importing

File: AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile (hunk 1 above)
Comment (Maxino22) [reply AliceFaberAcmeDemo/controllers/alice-controller/Dockerfile]: Initially experienced issues running it tested again and worked. I have removed that line.

**Review by amanji — APPROVED** [review]

**Pull request merged**